index.js

```diff
@@ -286,3 +309,47 @@ function toBuffer (chunk, encoding) {
     ? Buffer.from(chunk, encoding)
     : chunk
 }
+
+/**
+ * Most browsers send "br" (brotli) as the last value in
+ * in the 'Accept-Encoding' header which causes it to be
+ * deprioritized according to the spec.
+ *
+ * This is typically not what end users actually want so here
+ * we force the "br" (brotli) value to first in the list so that
+ * it will get properly prioritized and used.
+ *
+ * It's worth noting that although this is not "spec compliant",
+ * we belive it follows a well-established convention.
+ *
+ * @private
+ */
+function prioritize (str) {
+  return str && str.split(',')
+    .sort(sortEncodings)
+    .join(',')
+}
+
+/**
+ * Sort compression encodings in order of our preference:
+ * br > gzip > deflate
+ *
+ * @private
+ */
+function sortEncodings (a, b) {
+  if (a.indexOf('br') >= 0) {
```

Comment:
The encoding are case insensitive according to the spec, but this us treating them as case sensitive.

Reply:
Another great catch, I now lowercase the value of the header before I look for any specific values.

Reply:
This seems to now violate the spec in a different way: the `q` parameter name is case-sensitive, so `br;q=0` header would have `br` with a quality of 0, but `br;Q=0` would have a quality of 1. This change you made will now accidentally treat the latter as the former.

Reply:
I see what you are saying. I've updated the code so that this sorting function will sort the items in the header but it will not change their original case. It will also work with any casing of encoding values.

Reply:
Is there a reason why it is using `.indexOf` instead of just a comparison? I would think `.indexOf` would identify `bro` as `br` or similar. I'm not sure what that would all entail. I glanced again at the spec and this header doesn't allow random parameters to be specified, so it does seem unlikely to get mixed up in that way, but when looking, I realize that this header does accept the special `*` value. I know this sorting does not take that `*` into account there, so I wonder if that will cause issues with the logic? I tried a few different headers to see what the outcomes are and was curious if these make sense:

`*` : Results in `br` encoding
`*, gzip`: Results in `gzip` encoding
`gzip, *`: Results in `gzip encoding